pkgs/top-level/all-packages.nix

```diff
@@ -4550,6 +4550,8 @@ with pkgs;
 
   enca = callPackage ../tools/text/enca { };
 
+  engage = callPackage ../development/tools/engage { };
```

Comment:
This is not restricted to development. I believe it should be put on `tools/misc`.